h/static/scripts/session.js

```diff
@@ -156,7 +156,7 @@ function session($http, $resource, $rootScope, flash, raven, settings) {
 
   function process(data, headersGetter, status) {
     if (status < 200 || status >= 500) {
-      return;
+      return data;
```

Comment:
It turns out it doesn't matter what the return value is here because it is unused if the status is not 2xx, since the request is automatically retried in that case. For consistency with the previous code I've changed this to `return null`.